mirror/images.txt

```diff
@@ -7,7 +7,7 @@
 #
 
 # ceph-csi devel
-docker.io/rook/ceph:v1.9.8      rook/ceph:v1.9.8
+docker.io/rook/ceph:v1.10.0      rook/ceph:v1.10.0
```

Comment:
add a new block for ceph-csi v3.7, keep the older Rook version listed there

Reply:
My plan is to run Rook 1.10 even for 3.7 as we need to maintain 3.7 for a while and we need to run e2e on 3.7 branch on kubernetes 1.25

Reply:
ok, thanks for explaining!